WordPress/src/main/res/values-ar/strings.xml

```diff
@@ -31,88 +31,88 @@ Language: ar
+    <string name="gutenberg_native_double_tap_to_toggle_setting">انقر نقرًا مزدوجًا للتبديل إلى الإعدادات</string>
+    <string name="gutenberg_native_double_tap_to_select_an_image">انقر نقرًا مزدوجًا لتحديد صورة</string>
+    <string name="gutenberg_native_double_tap_to_select_a_video">انقر نقرًا مزدوجًا لتحديد فيديو</string>
+    <string name="gutenberg_native_double_tap_to_select">انقر نقرًا مزدوجًا للتحديد</string>
+    <string name="gutenberg_native_double_tap_to_remove_the_block">انقر نقرًا مزدوجًا لإزالة المكوِّن</string>
+    <string name="gutenberg_native_double_tap_to_redo_last_change">انقر نقرًا مزدوجًا لإعادة آخر تغيير</string>
+    <string name="gutenberg_native_double_tap_to_move_the_block_up">انقر نقرًا مزدوجًا لتحريك المكوِّن إلى أعلى</string>
+    <string name="gutenberg_native_double_tap_to_move_the_block_down">انقر نقرًا مزدوجًا لتحريك المكوِّن إلى أسفل</string>
+    <string name="gutenberg_native_double_tap_to_edit_this_value">انقر نقرًا مزدوجًا لتحرير هذه القيمة</string>
+    <string name="gutenberg_native_double_tap_to_change_the_value_using_slider">انقر نقرًا مزدوجًا لتغيير القيمة باستخدام شريط التمرير</string>
+    <string name="gutenberg_native_double_tap_to_add_a_block">انقر نقرًا مزدوجًا لإضافة مكوِّن</string>
+    <string name="gutenberg_native_double_tap_and_hold_to_edit">انقر نقرًا مزدوجًا مع الاستمرار للتحرير</string>
+    <string name="gutenberg_native_current_value_is_s">القيمة الحالية هي %s</string>
+    <string name="gutenberg_native_content">المحتوى…</string>
+    <string name="gutenberg_native_clear_all_settings">مسح كافة الإعدادات</string>
+    <string name="gutenberg_native_choose_from_device">الاختيار من جهاز</string>
+    <string name="gutenberg_native_annotations_sidebar">الشريط الجانبي للتعليقات التوضيحية</string>
+    <string name="gutenberg_native_an_unknown_error_occurred_please_try_again">حدث خطأ غير معروف. يُرجى المحاولة مرة أخرى.</string>
+    <string name="gutenberg_native_alt_text">النص البديل</string>
+    <string name="gutenberg_native_add_video">إضافة فيديو</string>
+    <string name="gutenberg_native_add_url">إضافة عنوان URL</string>
+    <string name="gutenberg_native_add_link_text">إضافة نص رابط</string>
+    <string name="gutenberg_native_add_image_or_video">إضافة صورة أو فيديو</string>
+    <string name="gutenberg_native_add_image">إضافة صورة</string>
+    <string name="gutenberg_native_add_block_here">إضافة مكوِّن هنا</string>
+    <string name="gutenberg_native_add_annotation">إضافة تعليق توضيحي</string>
+    <string name="gutenberg_native_add_a_description">إضافة وصف</string>
     <string name="reader_empty_saved_posts_content_description">أنقر فوق الزرّ \"حفظ المقالات\" لحفظ مقالةٍ في قائمتك.</string>
-    <string name="reader_acessibility_list_loaded">\"تم تحميل القائمة بعناصر %1$d.\"</string>
+    <string name="reader_acessibility_list_loaded">تم تحميل القائمة بعناصر %1$d.</string>
```

Comment:
I did not intend to modify strings that are not related to gutenberg. This was already in the last release https://github.com/wordpress-mobile/WordPress-Android/blob/release/13.7/WordPress/src/main/res/values/strings.xml#L1558 and it's hard to say (at least for me) if those quotes are intentional or not

Reply:
> I did not intend to modify strings that are not related to gutenberg.

Yep! You're right! I think we should only change the strings related to Gutenberg.